runtime/src/main/java/org/corfudb/runtime/view/TableRegistry.java

```diff
@@ -283,9 +297,8 @@ public static String getFullyQualifiedTableName(String namespace, String tableNa
     private <T extends Message> void addTypeToClassMap(T msg) {
         String typeUrl = getTypeUrl(msg.getDescriptorForType());
         // Register the schemas to schema table.
-        if (!classMap.containsKey(typeUrl)) {
-            classMap.put(typeUrl, msg.getClass());
-        }
+        ((ProtobufSerializer)Serializers.getSerializer(ProtobufSerializer.PROTOBUF_SERIALIZER_CODE))
+                .getClassMap().put(typeUrl, msg.getClass());
```

Comment:
Do you still need this change? Since the classMap is already pointing to the old classMap in the constructor, so you can directly operate on the classMap, right?